Formula/apache-brooklyn-cli.rb

```diff
@@ -0,0 +1,62 @@
+require "language/go"
+
+class ApacheBrooklynCli < Formula
+  desc "Apache Brooklyn command-line interface"
+  homepage "https://brooklyn.apache.org"
+  url "https://github.com/apache/brooklyn-client.git",
+      :tag => "rel/apache-brooklyn-0.9.0",
+      :revision => "bc8593a933fcb76327ae4a511643e39d25a87ba2"
```

Comment:
Can we use [the release tarball](https://github.com/apache/brooklyn-client/archive/rel/apache-brooklyn-0.9.0.tar.gz) instead of directly cloning from the repository? A single-file download is more resource-friendly towards both GitHub and users that build from source. It also allows us to check the integrity of the download (you'll need to add a `sha256` statement).